rasa/core/lock_store.py

```diff
@@ -247,14 +247,14 @@ def _create_from_endpoint_config(
     elif endpoint_config.type == "redis":
         lock_store = RedisLockStore(host=endpoint_config.url, **endpoint_config.kwargs)
     else:
-        lock_store = _load_from_module_string(endpoint_config)
+        lock_store = _load_from_endpoint_config(endpoint_config)
 
     logger.debug(f"Connected to lock store '{lock_store.__class__.__name__}'.")
 
     return lock_store
 
 
-def _load_from_module_string(endpoint_config: EndpointConfig) -> "LockStore":
+def _load_from_endpoint_config(endpoint_config: EndpointConfig) -> "LockStore":
     """Given the name of a `LockStore` module tries to retrieve it."""
```

Comment:
"Given the name" sounds a bit like the name would be the argument of the function. Let's change it to match the doc strings in the other modules.
```suggestion
    """Retrieve a `LockStore` based on its class name."""
```